pipelines/batch/Dockerfile

```diff
@@ -12,13 +12,23 @@
 # See the License for the specific language governing permissions and
```

Comment:
I am not sure why this is here; can you please replace it with the usual license text?

Reply:
What is the ususal license text? I took a look at the files in the repo, and all of them had the same header.

Reply:
Sorry my bad; GitHub hid the rest of the text and I did not notice!